samples/dynamic_alloc/main.cpp

```diff
@@ -49,7 +49,7 @@ bool DetourTransaction(std::function<bool()> callback) {
   LONG status = DetourTransactionBegin();
   if (status != NO_ERROR) {
     Log("DetourTransactionBegin failed with %08x\n", status);
-    return status;
+    return status == NO_ERROR;
```

Comment:
I think we should just return `false` explicitly, the current suggested change can potentially raise a different static analysis warning, since the comparison is guaranteed to always be false.

Reply:
I can fix later, I'll just merge as is... :)